neo.UnitTests/UT_UIntBenchmarks.cs

```diff
@@ -0,0 +1,333 @@
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Diagnostics;
+using System;
+//using System.Runtime.CompilerServices.Unsafe;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_UIntBenchmarks
+    {
+        int MAX_TESTS = 1000000; // 1 million
+
+        byte[][] base_32_1;
+        byte[][] base_32_2;
+        byte[][] base_20_1;
+        byte[][] base_20_2;
+
+        private Random random;
+
+        [TestInitialize]
+        public void TestSetup()
+        {
+            int SEED = 123456789;
+            random = new Random(SEED);
+
+            base_32_1 = new byte[MAX_TESTS][];
+            base_32_2 = new byte[MAX_TESTS][];
+            base_20_1 = new byte[MAX_TESTS][];
+            base_20_2 = new byte[MAX_TESTS][];
+
+            for(var i=0; i<MAX_TESTS; i++)
```

Comment:
@igormcoelho did you calculate the amount of RAM used for this test? is not excessive?